boards/unphone9.json

```diff
@@ -0,0 +1,55 @@
+{
+  "build": {
+    "arduino":{
+      "ldscript": "esp32s3_out.ld"
+    },
+    "core": "esp32",
+    "extra_flags": [
+      "-DUNPHONE_SPIN=9",
+      "-DARDUINO_USB_CDC_ON_BOOT=0",
+      "-DARDUINO_RUNNING_CORE=1",
+      "-DARDUINO_EVENT_RUNNING_CORE=1"
+    ],
+    "f_cpu": "240000000L",
+    "f_flash": "80000000L",
+    "flash_mode": "dio",
+    "hwids": [
+      [
+        "0x16D0",
+        "0x1178"
+      ]
+    ],
+    "mcu": "esp32s3",
+    "variant": "unphone9"
```

Comment:
Same as for `unPhone 8`. There is no such variant https://github.com/espressif/arduino-esp32/tree/master/variants

Reply:
See above, a PR is pending on the core repo.